.yeoman/src/get-existing/index.test.js

```diff
@@ -0,0 +1,33 @@
+const getExistingPackages = require('./index');
+const fs = require('fs');
+
+jest.mock('fs', () => ({
+  readdirSync: jest.fn(),
+  statSync: jest.fn(),
+}));
+
+describe('getExistingPackages', () => {
+  it('should return a list of exisiting package names', async () => {
+    const componentDir = [
+      'foobar.js',
+      'README.md',
+      'psammead-foobar',
+      'psammead-foobar',
+    ];
+
+    fs.readdirSync.mockReturnValue(componentDir);
+
+    fs.statSync.mockImplementation(path => {
+      return {
+        isDirectory: () => path.includes('psammead-'),
+      };
+    });
+
+    const exisitingPackages = getExistingPackages();
+
+    expect(exisitingPackages).toEqual(expect.not.stringMatching('.'));
```

Comment:
Should this not be `stringContaining`? also feels a bit redundant given the next `expect` assertion

Reply:
I have removed this test and updated the below test to have `stringContaining` 👍 thanks :)